cscs-checks/tools/profiling_and_debugging/ipm_mpi_omp.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.required_version('>=2.14')
+@rfm.parameterized_test(['C'], ['C++'], ['F90'])
+class Ipm(rfm.RegressionTest):
+    def __init__(self, lang):
+        super().__init__()
+        self.name = 'IPM_%s' % lang.replace('+', 'p')
+        self.descr = '%s check' % lang
+        self.language = lang
+        self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc']
+        self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu',
+                                    'PrgEnv-intel', 'PrgEnv-pgi']
+        self.prgenv_flags = {
+            'PrgEnv-gnu': ['-O2 -g -fopenmp'],
```

Comment:
Better use `['-O2', '-g', '-fopenmp']`  and do so with the rest of the dictionary.